modules/core/src/webgl/vertex-array.js

```diff
@@ -313,10 +310,12 @@ export default class VertexArray {
   // Updates all constant attribute values (constants are used when vertex attributes are disabled).
   // This needs to be done repeatedly since in contrast to buffer bindings,
   // constants are stored on the WebGL context, not the VAO
-  _setConstantAttributes(vertexCount) {
+  _setConstantAttributes(vertexCount, instanceCount) {
+    // TODO - use accessor to determine what length to use
+    const elementCount = Math.max(vertexCount | 0, instanceCount | 0);
```

Comment:
> Nit: should we have an assert instead of converting them to 0?

Yes, will improve this after release. For now I just wanted to minimize the risk of not rendering in the web page.